model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/metrics.proto

```diff
@@ -517,6 +517,10 @@ message MonitoringInfoTypeUrns {
     //   - sum:   beam:coder:varint:v1
     //   - min:   beam:coder:varint:v1
     //   - max:   beam:coder:varint:v1
+    //
+    // Note that when count is zero, the SDK may or may not send sum, min, and
```

Comment:
"may" in specs usually implies the conditional, so we can avoid the "or may not". I think "omit" here makes it clearer. Same for the paired paragraph below.

```suggestion
    // Note that when count is zero, the SDK may omit sum, min, and
```